src/components/history/messages/regular/task/CreateTask.js

```diff
@@ -0,0 +1,146 @@
+import React, {PureComponent, PropTypes} from 'react'
+import injectSheet from 'grape-web/lib/jss'
+import {
+  FormattedMessage,
+  defineMessages,
+  intlShape,
+  injectIntl
+} from 'react-intl'
+import {Input} from 'material-ui/Input'
+import {FormGroup, FormControl, FormLabel} from 'material-ui/Form'
+import List from 'material-ui/List'
```

Comment:
not a big deal but I think that we should stick with one import style.. 
either `import {List}` or `import List` mui provides both, that's why for example the `import {Input}` above works